jwst/scripts/world_coords.py

```diff
@@ -324,36 +337,35 @@ def imaging_coords(model):
     """
     hdulist = fits.HDUList()
     phdu = fits.PrimaryHDU()
-    phdu.header['filename'] = model.meta.filename
-    phdu.header['data'] = 'world coordinates'
+    phdu.header["filename"] = model.meta.filename
+    phdu.header["data"] = "world coordinates"
     hdulist.append(phdu)
 
     output_frame = model.available_frames[-1]
     bb = model.meta.wcs.bounding_box
     x, y = wcstools.grid_from_bounding_box(bb, step=(1, 1), center=True)
-    ra, dec, lam = slit(x + 1, y + 1)
+    ra, dec, lam = slit(x + 1, y + 1)  # noqa
     world_coordinates = np.array([lam, ra, dec])
 
     imhdu = fits.ImageHDU(data=world_coordinates)
-    imhdu.header['PLANE1'] = 'lambda, microns'
-    imhdu.header['PLANE2'] = '{0}_x, deg'.format(output_frame)
-    imhdu.header['PLANE3'] = '{0}_y, deg'.format(output_frame)
-    imhdu.header['SLIT'] = "SLIT_{0}".format(i)
+    imhdu.header["PLANE1"] = "lambda, microns"
+    imhdu.header["PLANE2"] = "{0}_x, deg".format(output_frame)
+    imhdu.header["PLANE3"] = "{0}_y, deg".format(output_frame)
+    imhdu.header["SLIT"] = "SLIT_{0}".format(i)  # noqa
```

Comment:
I added these two `#noqa` comments because it appears that `slit()` and `i` are both undefined, and I couldn't find where they are supposed to come from.